TSC-Charter.md

```diff
@@ -138,16 +138,29 @@ item has appeared to reach a consensus the moderator will ask "Does anyone
 object?" as a final call for dissent from the consensus.
 
 If an agenda item cannot reach a consensus a TSC member can call for
-either a closing vote or a vote to table the issue to the next meeting.
-The call for a vote must be seconded by a majority of the TSC or else the
-discussion will continue.
+either a closing vote or a vote to table the issue to the next meeting or to
+be conducted asynchronously through a system to be chosen and seconded by the
+majority of TSC members. The call for a vote must be seconded by a majority
+of the TSC or else the discussion will continue.
 
 For all votes, a simple majority of all TSC members for, or against, the issue
 wins. A TSC member may choose to participate in any vote through abstention.
 
 Note that, in addition to requiring a simple majority vote of the TSC, all
 changes to this charter are also subject to approval from the CPC.
 
+### Section 7.1. Emergency Votes
```

Comment:
My personal feeling is it might be better to just allow votes to be called through the mailing list.  I'm not sure a meeting is needed, and the mailing is likely to reach more TSC members.

Reply:
+1 to making a clear process for calling a vote without requiring a meeting rather than making a specific carve out. Honestly if one TSC member calls the vote and there are no objections within a 24 hour period that could be reasonable enough to kick one off. Waiting for a majority of the TSC to +1 could end up causing a delay long enough that we have the next meetings ready to go already 😂

Reply:
So instead of having two ways of calling for vote and executing a vote, we only have one via mailing list (potentially via issues for votes that can be public)? I think that's a good idea, it allows for input from any TSC members (not only those who live in a timezone favourable for a particular meeting) and allow us to vote faster.

Reply:
@MylesBorins +100000, there's no reason to wait for 10 +1s via email to start the vote process, 24h with no one objecting to having a vote seems reasonable.

Reply:
If someone doesn't like the vote they can vote no 😂

Reply:
I agree! I would like to see the need for calling a vote in a meeting or with a majority removed.